tuf/keys.py

```diff
@@ -832,6 +848,17 @@ def verify_signature(key_dict, signature, data):
       else:
         valid_signature = tuf.pycrypto_keys.verify_rsa_signature(sig, method,
                                                                  public, data) 
+    elif _RSA_CRYPTO_LIBRARY == 'pyca-cryptography': 
+      if 'pyca-cryptography' not in _available_crypto_libraries: # pragma: no cover
+        message = 'Metadata downloaded from the remote repository specified' +\
+          ' an RSA signature.  Verifying RSA signatures requires pyca/cryptography.' +\
+          '\n$ pip install cryptography, or pip install tuf[tools].'
+        raise tuf.UnsupportedLibraryError(message)
```

Comment:
Error message improved [here in PR #311](https://github.com/theupdateframework/tuf/pull/311/files#r52197784). I don't love how it's structured, still, but I haven't yet grokked enough to be confident in that. (What is determining which library to use? User selection in config? Will see.)

TODO.